packages/errors/src/json-rpc-error.ts

```diff
@@ -22,7 +23,7 @@ import { safeCaptureStackTrace } from './stack-trace';
 import { getSolanaErrorFromTransactionError } from './transaction-error';
 
 interface RpcErrorResponse {
-    code: bigint | number;
+    code?: bigint | number;
```

Comment:
I think we should probably leave the type alone (ie. it should conform to the JSON-RPC spec) but then jam a bunch of assertions into the `get*` method that produce a `_MALFORMED_` error when they fail.